optuna/logging.py

```diff
@@ -102,6 +102,32 @@ def get_verbosity() -> int:
 def set_verbosity(verbosity: int) -> None:
     """Set the level for the Optuna's root logger.
 
+    Example:
+
+        Set the logging level ``optuna.logging.WARNING``.
+
+        .. testsetup::
+
+            def objective(trial):
+                x = trial.suggest_int("x", -10, 10)
+                return x ** 2
+
+        .. testcode::
+
+            import optuna
+
+            # There are INFO level logs.
+            study = optuna.create_study()
+            study.optimize(objective, n_trials=10)
+            # [I 2021-10-31 02:59:35,088] Trial 0 finished with value: 16.0 ...
+            # [I 2021-10-31 02:59:35,091] Trial 1 finished with value: 1.0 ...
+            # [I 2021-10-31 02:59:35,096] Trial 2 finished with value: 1.0 ...
+
+            # Set the logging level WARNING.
+            # There are INFO level logs.
```

Comment:
```suggestion
            # Setting the logging level WARNING, the INFO logs are suppressed.
```

Reply:
I think `There are INFO level logs.` is not correct here.

Reply:
Thank you comments. I accept the suggestion.